helix-core/src/test/java/org/apache/helix/task/assigner/TestAssignableInstance.java

```diff
@@ -55,18 +55,20 @@ public void testInvalidInitialization() {
 
   @Test
   public void testInitializationWithQuotaUnset() {
+    int testCurrentTaskThreadPoolSize = 100;
```

Comment:
Nit: rename to "expectedCurrentTaskThreadPoolSize"?

Reply:
Sure.